### PR TITLE
Changed menu from horizontal to vertical

### PR DESCRIPTION
**Pull request by 4i60r**

### CODE REVIEW

File: src/ralph/settings/base.py [hunk 1]
```diff
@@ -131,7 +131,6 @@ def str_to_bool(s: str) -> bool:
     'sql_mode': 'TRADITIONAL',
     'charset': 'utf8',
     'init_command': """
-    SET storage_engine=INNODB;
```
Comment (mkurek) [inline src/ralph/settings/base.py:134]: revert it (and apply it in local settings if you have to)

**Comment by szok** [comment]
please add screenshots.

**Comment by 4i60r** [comment]
screenshot from 2016-08-09 15-18-52
screenshot from 2016-08-09 15-19-54

File: src/ralph/static/src/scss/components/tabs.scss [hunk 2]
```diff
@@ -47,3 +47,64 @@
...
+    padding-right: 0;
+}
+.no-right-padding {
+    padding-right: 0;
+}
+.no-left-padding {
+    padding-left: 0;
+}
+#content-main .content {
+    min-height: 364px;
```
Comment (mkurek) [inline src/ralph/static/src/scss/components/tabs.scss]: what is this magic value? :)

**Comment by ar4s** [comment]
`src/ralph/admin/templates/admin/change_form.html` change this file either 
`no-right-padding` -> `no-horizontal-padding`
`large-2` -> `large-1` etc.

**Comment by ar4s** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**